usagov/usagov-2021
# USAGOV-2099: fix git tag creator and docs

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2099

## Description


## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

1. During the "CloudGov Deploy CMS" step of the CircleCI deployment, the instructions output to the build log for creating an annotated git tag should contain all 3 digest strings (waf, cms and www).  The line will begin with `bin/deploy/git-annotated-tag-creator.sh`

2. When running the command `bin/deploy/git-annotation-parser.sh` the output should show the variable name PROTECTED_APP_NAMES (plural, not singular) on the output instructions for the waf.


## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
